generator_test.go

```diff
@@ -376,6 +378,103 @@ func testNewV5DifferentNamespaces(t *testing.T) {
 	}
 }
 
+func Test_defaultHWAddrFunc(t *testing.T) {
```

Comment:
I don't like this underscore very much, because no other top level tests have one. Indeed, `TestdefaultHWAttrFunc` does not look nice, but I wouldn't mind `TestDefaultHWAddrFunc`, even though the token after `Test` does not match the function being tested exactly.

I don't feel strongly about this at all, so we can leave it as-is if you'd like.

Reply:
Agree with @acln0 , Let's have `TestDefaultHWAddrFunc`